include/graph/graph.h

```diff
@@ -0,0 +1,38 @@
+#ifndef GRAPH_H
+#define GRAPH_H
+
+#include <unordered_map>
+#include <vector>
+
+#include "./layer/layer.h"
+#include "./tensor/tensor.h"
+
+class Graph {
+ private:
+  std::unordered_map<int, Layer*> layers_;
+  Tensor<double> inputTensor_;
+  Tensor<double>* outputTensor_;
+  int start_ = -1;
+  int end_ = -1;
+  bool bfs_helper(int start, int vert, bool flag,
+                  std::vector<int>* v_ord) const;
+
+ public:
+  Graph();
+
+  void addLayer(Layer& lay);
+  void addEdge(Layer& layPrev, Layer& layNext);
```

Comment:
```suggestion
  bool addLayer(Layer& lay, std::vector<LayerID>& InputsLayers, std::vector<LayerID>& OutputLayers);
```
bool for check - Layer is added or not
addEdge is redudant